### PR TITLE
🐛 installments: withdraw modal MAX crash

### PR DESCRIPTION
**Pull request by alexx855**
This in an attempt to fix the issue #1391

### CODE REVIEW

File: hooks/useInstallmentsData.ts [hunk 1]
```diff
@@ -45,6 +45,16 @@ export default function useInstallmentsData({
         irmParameters,
         timestamp,
       ] as const;
+      // Pevents division by zero error in @exactly/lib/esm/installments/split function if there are no fixed pools
+      if (fixedPoolsUtilizations.length === 0) return;
+      // Check for already matured pools, @exactly/lib/esm/installments/split will throw an error if the pool is already matured
+      if (
```
Comment (sebipap) [inline hooks/useInstallmentsData.ts:51]: prefer not adding comments and naming the variable inside the `if` statement.
```
if (alreadyMatured) { ... }
```

**Review by sebipap — COMMENTED** [review]
LGTM! 😊
Thanks for contributing, just adding a small code-style suggestion.
Avoid code comments.

File: hooks/useInstallmentsData.ts (hunk 1 above)
Comment (alexx855) [reply hooks/useInstallmentsData.ts]: Hello @sebipap! code style fixed 😊

**Comment by franm91** [comment]
Alex, tyvm for the contribution; this issue occurs when the installment hook tries to calculate installments when the operation is not a borrowAtMaturity (ej: withdrawal); we are sending a fix that only calculates the installments when the operation is the correct one.

**Comment by alexx855** [comment]
Awesome, it's not crashing anymore after that fix.

Un saludo 🫡

> Alex, tyvm for the contribution; this issue occurs when the installment hook tries to calculate installments when the operation is not a borrowAtMaturity (ej: withdrawal); we are sending a fix that only calculates the installments when the operation is the correct one.

**Pull request closed without merging**